### PR TITLE
Fixed broken local deployment

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### Description
This PR fixes broken local deployment (`make deploy`)

/cc @periklis 
/assign @periklis

### CODE REVIEW

File: olm_deploy/scripts/catalog-build.sh [hunk 1]
```diff
@@ -16,5 +16,5 @@ if [ -n ${LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY} ] ; then
         echo "Unexpected message from oc port-forward: $PORT_FORWARD_STDOUT"
     fi
 fi
-echo "Pushing image ${IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}"
-podman push  --tls-verify=false ${IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}
+echo "Pushing image ${LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}"
+podman push  --tls-verify=false ${LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}
```
Comment (periklis) [inline olm_deploy/scripts/catalog-build.sh:20]: This looks weird, we should always use `IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY` and use `LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY` only when we set the former elsewhere.

**Comment by igor-karpukhin** [comment]
@periklis are we okay with this PR?

**Pull request merged**